File: c-c/7.8.cpp
//7-8 超速判断
/*
2021.10.09
20.03.20
By Xcl
*/
#include <stdio.h>

int main()
{
    int a;
    scanf("%d",&a);
    if(a>60) printf("Speed: %d - Speeding",a);
    else printf("Speed: %d - OK",a);
    return 0;
}

File: java-i/7.8.java
//7-8 超速判断
/*
2021.10.09
20.08.06
By Xcl
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a=sc.nextInt();
        if(a>60) System.out.print("Speed: "+a+" - Speeding");
        else System.out.print("Speed: "+a+" - OK");
    }
}
